test/integration/admin_crud_specialties_test.rb

```diff
@@ -39,4 +39,29 @@ class AdminCrudSpecialtiesTest < ActionDispatch::IntegrationTest
     assert page.has_content? specialty.name
     assert page.has_css? "#database-image"
   end
+
+  test "admin edits a specialty" do
+    admin = create(:admin)
+    specialty = create(:specialty)
+    ApplicationController.any_instance.stubs(:current_admin).returns(admin)
+
+    visit 'admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.